fsspec/implementations/tests/test_sftp.py

```diff
@@ -2,6 +2,7 @@
 import subprocess
 import time
 from tarfile import TarFile
+import os
 
```

Comment:
```suggestion
import os
import subprocess
import time
from tarfile import TarFile

```

Reply:
Actually, should be one line higher above shlex, but the UI here won't let me do that!